provider/provider.go

```diff
@@ -256,15 +255,6 @@ func configureDatabricksClient(ctx context.Context, d *schema.ResourceData) (int
 		authsUsed["config profile"] = true
 		pc.ConfigFile = v.(string)
 	}
-	if _, ok := d.GetOk("basic_auth"); ok {
-		authsUsed["password"] = true
-		username, userOk := d.GetOk("basic_auth.0.username")
-		password, passOk := d.GetOk("basic_auth.0.password")
-		if userOk && passOk {
-			pc.Username = fmt.Sprintf("%s", username)
-			pc.Password = fmt.Sprintf("%s", password)
-		}
-	}
```

Comment:
the `basic_auth` is still mentioned in the `docs/resources/token.md` (in example) and in `docs/index.md` in phrase "when you don't specify host/token/basic_auth/azure attributes"